src/components/Personal-Zone/Searcher/AddResumeDialog/ProgressBar/Steps/PreviousJobs.tsx

```diff
@@ -187,9 +187,10 @@ const PreviousJobsStep: React.FC<PreviousJobsStepProps> = (props): JSX.Element =
                         setCurrentId((prevCurrentId) => prevCurrentId + 1);
                         setEnteredPrevJob(defaultJob);
                     }}
-                    disabled={previousJobs.length === MAX_NUM_PREVIOUS_JOBS_EXPERIENCE}>
+                    disabled={previousJobs.length === MAX_NUM_PREVIOUS_JOBS_EXPERIENCE - 1}>
```

Comment:
are you sure with the `-1`, that will not give us only 2 previousJobs?

Reply:
No, because the previous jobs do not include the the default first one